lumicks/pylake/channel.py

```diff
@@ -62,6 +62,11 @@ def timestamps(self):
         """Absolute timestamps (since epoch) which correspond to the channel data"""
         return self._src.timestamps
 
+    @property
+    def time(self):
+        """Relative time (in seconds) that corresponds to the channel data"""
+        return (self._src.timestamps-self._src.timestamps[0])/1e9
```

Comment:
I would suggest using `.start` instead of `.timestamps[0]`. The reason for that can be found in `Continuous.timestamps`: it generates a new `np.arange()` every time that it's called. That means that it would create two identical arrays one after the other. That can be very wasteful and slow for large datasets. On the other hand, `.start` is fast for all 3 channel kinds.

I would also suggest multiplying by `1e-9` rather than dividing by `1e9`. Again, for large datasets, multiplication is pretty much always faster than division. I would not say that you should go out of your way to always multiply instead of divide but this is a nice default when possible.
```suggestion
        return (self._src.timestamps - self._src.start) * 1e-9
```